Dockerfile

```diff
@@ -9,16 +9,23 @@ ENV MNEMONIC_TEST_ACCOUNT="abandon abandon abandon abandon abandon abandon aband
 # Create app directory
 WORKDIR /usr/src/app
 
-COPY package*.json ./
 RUN apk add --update --no-cache git
+RUN apk add --update python make g++\
```

Comment:
what does updating python do?

Reply:
it just adds python which is a requirement for node-gyp - basically I needed it to run it locally